torch/_dynamo/eval_frame.py

```diff
@@ -761,12 +768,17 @@ def graph_with_interpreter(*args):
             with torch.fx.traceback.preserve_node_meta():
                 return torch.fx.Interpreter(graph).run(*args)
 
-        graph = make_fx(
-            graph_with_interpreter,
-            decomposition_table=decomposition_table,
-            tracing_mode=tracing_mode,
-            _allow_non_fake_inputs=True,
-        )(*graph_captured_input)
+        fake_tensor_mode = null_context()
+        if example_fake_inputs is not None and len(example_fake_inputs) > 0:
+            fake_tensor_mode = example_fake_inputs[0].fake_mode
```

Comment:
nit: it is better if you follow the aot autograd pattern exactly, which is to iterate over the fake inputs and accept the first one that actually has a fake mode. This handles the case when the first input is not actually a tensor (if that could occur)